shell/client/admin/stats.html

```diff
@@ -159,6 +159,9 @@
       Any stats shared are submitted anonymously.
     </p>
     <p>
+      {{#if featureKeyForceStats}}
+        You have a Sandstorm for Work trial key, and as part of that agreement, you elected to opt into stats reporting. Please upgrade to a paid key if you wish to disable this.
```

Comment:
Let's change this to:

> During your free trial of Sandstorm for Work, the stats on this page will be reported to Sandstorm so that we can help you get up and running. Once you purchase a key (or downgrade to standard Sandstorm by removing your trial key), these stats will stop being reported.

Additionally, in trial mode we should not show the heading "Publish anonymously to Sandstorm dev team" since trial stats are not anonymous. Let's replace the whole section. Maybe we should put it in a highlighted box?